libraries/chain/controller.cpp

```diff
@@ -1117,6 +1117,13 @@ struct controller_impl {
       return prev->block_num();
    }
 
+   // returns 0 for legacy
```

Comment:
I wonder if this is guaranteed by the standard?

Reply:
Yes, https://en.cppreference.com/w/cpp/language/zero_initialization